index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+   <div class="head">
+    <a href="#" class="link"><div class="logo">My Logo</div></a>
+   </div>
+   <nav class="menubar">
+    <img src="Media/Union.svg" class="menu"></div>
+  </header>  
+  <section class="headline">
```

Comment:
- `[OPTIONAL]` Kindly please try to wrap your main content inside the `main` tag because it will make your code `semantically` more correct and help your portfolio to have better accessibility. Please look at the below example.

```suggestion
  <main>
     <section class="headline">
     .....
     ....
     ...
     ..
     .
     </section>
  </main>
```